R/api.R

```diff
@@ -421,18 +487,18 @@ get_comments <- function(item) {
 #' @title Hacker News nested comments with root comment
 #'
 #' @description Returns specified item and all comments under it. Comments are retrieved
-#' using Hacker News api
+#' using the Hacker News API
 #'
 #' @param item item whose children (comments) will be retrieved
 #'
 #' @return dataframe containing specified item and all comments under that item
 #'
-#'
 get_comments_with_root <- function(item) {
   if (is.null(item$kids)) {
     comment_to_dataframe_row(item)
   } else {
     kids <- hackeRnews::get_items_by_ids(item$kids)
+    kids <- kids[!is.na(kids)]
```

Comment:
not sure whether it is the best choice to remove all kids instead of returning id and rest of the data as `NA`. User can easily filter those rows.  But if that's the decision of all team I'm ok with that.

Reply:
To be honest I am still undecided on this one. Because on the one hand returning an id of a "broken"  item might result in users trying to operate on it on the other hand the user might want to know how many comments are there in a single story which means that removing the data results in loss of information.